server/.env.dev

```diff
@@ -11,6 +11,9 @@ MP_123_SECRET_KEY=
 JWT_SECRET_KEY=secret
 PROXY_URL=http://localhost:5001
 TEST_CLIENT_BACKEND_URL=localhost:8000
+MAILER_SEND_API_KEY=mlsn.c6b77816e68ab3483699cf354f722da3bb6f14256ce856603987c627cf67dde2
+MAILER_SEND_TEMPLATE_ID=v69oxl517mz4785k
+LAYER8_EMAIL_DOMAIN=trial-k68zxl2eyo9lj905.mlsender.net
```

Comment:
please revert this commit then do force push a new commit without leaking your key here, its critical security issue